### PR TITLE
resource/alicloud_oss_bucket_replication: Fix rule-id bug.

### PR DESCRIPTION
**Pull request by huiguangjun**
…eader.

### CODE REVIEW

File: alicloud/resource_alicloud_oss_bucket_replication.go [hunk 1]
```diff
@@ -146,7 +147,6 @@ func resourceAlicloudOssBucketReplication() *schema.Resource {
 			},
 			"progress": {
 				Type:     schema.TypeList,
-				Optional: true,
```
Comment (ChenHanZhang) [inline alicloud/resource_alicloud_oss_bucket_replication.go:149]: 为啥去掉了Optional

File: website/docs/r/oss_bucket_replication.html.markdown [hunk 2]
```diff
@@ -13,7 +13,7 @@ Provides an independent replication configuration resource for OSS bucket.
 
 For information about OSS replication and how to use it, see [What is cross-region replication](https://www.alibabacloud.com/help/doc-detail/31864.html) and [What is same-region replication](https://www.alibabacloud.com/help/doc-detail/254865.html).
 
--> **NOTE:** Available in v1.161.0+.
+-> **NOTE:** Available since v1.161.0+.
```
Comment (ChenHanZhang) [inline website/docs/r/oss_bucket_replication.html.markdown:16]: 去掉+号

Comment (huiguangjun) [reply website/docs/r/oss_bucket_replication.html.markdown]: 已修改

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**